packages/application-extension/src/index.tsx

```diff
@@ -286,6 +287,7 @@ const downloadPlugin: JupyterFrontEndPlugin<void> = {
             if (item.type === 'directory') {
               return;
             }
+            await contents.save(item.path);
```

Comment:
This corresponds to the following case:

![image](https://user-images.githubusercontent.com/591645/166639913-d06c7f68-d9f1-4f91-ab0b-9ab7c1da2b31.png)

And the UI tests are failing because the second parameter (content) is not specified and hence storing `null`.

Reply:
Maybe we can omit this case for now, and only save when using the docmanager command?

Reply:
That would mean:

```suggestion
```

Reply:
Ah that's what that means. Sure, let's go for that.